Cargo.toml

```diff
@@ -26,25 +26,28 @@ incremental = true
 [workspace.dependencies]
 # Eth deps
 alloy-rlp = { version = "0.3.4", default-features = false }
-ef-tests = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0-rc.1", default-features = false, features = [
+ef-tests = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = false, features = [
   "ef-tests",
 ] }
 ethers-signers = { git = "https://github.com/gakonst/ethers-rs.git", tag = "v2.0" }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0-rc.1", default-features = false }
-revm-interpreter = { git = "https://github.com/bluealloy/revm", rev = "a28a543", default-features = false }
-revm-primitives = { git = "https://github.com/bluealloy/revm", rev = "a28a543", default-features = false }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v1.0.0", default-features = true, features = [
+  "std",
+] }
+revm-interpreter = { version = "6.0.0", default-features = false }
+revm-primitives = { version = "5.0.0", default-features = false }
 
 # Starknet deps
-cairo-lang-casm = { git = "https://github.com/kkrt-labs/cairo.git", branch = "v2.5.4", default-features = false }
-cairo-lang-starknet = { git = "https://github.com/kkrt-labs/cairo.git", branch = "v2.5.4", default-features = false }
-cairo-lang-utils = { git = "https://github.com/kkrt-labs/cairo.git", branch = "v2.5.4", default-features = false }
-cairo-vm = { git = "https://github.com/kkrt-labs/cairo-vm.git", branch = "v0.9.2", features = [
-  "print",
+cairo-lang-casm = "2.6.3"
+cairo-lang-starknet = "2.6.3"
+cairo-lang-utils = "2.6.3"
+cairo-lang-starknet-classes = "2.6.3"
+cairo-vm = "0.9.2"
```

Comment:
anyway we can get https://github.com/lambdaclass/cairo-vm/blob/v1.0.0-rc0/vm/Cargo.toml ? This contains the print feature. If not we can still proceed since I don't think Eni uses the print feature a lot these days.

Reply:
as discussed with @enitrat, we will merge this as the printing isn't necessary right now and it is preferred to have the prod release instead of the dev release.